clearcut_detection_backend/frontend/src/components/Calendar/Calendar.js

```diff
@@ -1,18 +1,73 @@
-import React, { PureComponent } from 'react';
+import React, { Component } from 'react';
 import 'react-dates/initialize';
 import 'react-dates/lib/css/_datepicker.css';
 import moment from 'moment';
+import { debounce } from 'lodash';
+import SidebarTitle from '../SidebarTitle';
 import { DateRangePicker, isInclusivelyAfterDay } from 'react-dates';
+
 import './Calendar.css';
 
-export default class Calendar extends PureComponent {
+export default class Calendar extends Component {
+  state = {
+    daySize: null,
+    numberOfMonths: null
+  };
+
+  handleResize = () => this.changeCalendarSize();
+
+  changeCalendarSize = () => {
+    window.innerWidth < 768
+      ? this.setState({ numberOfMonths: 1, daySize: 32 })
+      : this.setState({ numberOfMonths: 2, daySize: 38 });
+  };
+
+  componentDidMount() {
+    this.changeCalendarSize();
+
+    window.addEventListener('resize', debounce(this.handleResize, 300));
+  }
```

Comment:
?

Reply:
This block is needed to change calendar props by device resolution.